src/snapred/backend/recipe/CalibrationNormalizationRecipe.py

```diff
@@ -0,0 +1,66 @@
+import json
+from typing import Any, Dict
+
+from snapred.backend.dao.ingredients import NormalizationCalibrationIngredients as Ingredients
+from snapred.backend.log.logger import snapredLogger
+from snapred.backend.recipe.algorithm.CalibrationNormalizationAlgo import CalibrationNormalizationAlgo
+from snapred.backend.recipe.algorithm.WashDishes import WashDishes
+from snapred.meta.decorators.Singleton import Singleton
+
+logger = snapredLogger.getLogger(__name__)
+
+
+@Singleton
+class CalibrationNormalizationRecipe:
+    def __init__(self):
+        pass
+
+    def chopIngredients(self, ingredients: Ingredients):
+        self.runNumber = ingredients.reductionIngredients.runConfig.runNumber
+        self.backgroundRunNumber = ingredients.backgroundReductionIngredients.runConfig.runNumber
+
+    def unbagGroceries(self, groceryList: Dict[str, Any]):
+        """
+        Checkout the workspace names needed for this recipe.
+        It is necessary to provide the followign keys:
+         - "InputWorkspace": the raw vanadium data
+         - "BackgroundWorkspace": the background raw vanadium data
+         - "Ingredients": requirements for sub algo calls
+        """
+
+        self.rawInput = groceryList["inputWorkspace"]
+        self.rawBackgroundInput = groceryList["backgroundWorkspace"]
+        self.groupingWS = groceryList["groupingWorkspace"]
+        self.outputWS = groceryList.get("outputWorkspace", "")
```

Comment:
Worth pointing out, if `"outputWorkspace"` is not present in the grocery list, then the algorithm will fail validation